src/cards/ha-media_player-card.js

```diff
@@ -279,6 +279,14 @@ class HaMediaPlayerCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
     }
 
     // We have a new picture url
+    // If entity picture is non-relative, we use that url directly.
+    if (picture.substr(0, 1) !== "/") {
```

Comment:
This seems kinda smelly... would it be better to add `media_image_remotely_accessible` to state attributes so we can key off that directly?

Reply:
I agree that it is a little fragile. This implementation is directly tied to how the media player is implemented. If album art is not externally accessible, we set it to a relative url that points at the media player album art proxy. (which sucks btw, because it means we change states every 30s for no good reason)